applications/RANSApplication/custom_utilities/rans_check_utilities.h

```diff
@@ -4,10 +4,10 @@
 //   _|\_\_|  \__,_|\__|\___/ ____/
```

Comment:
I think these utilities can be removed by using what I posted above.

Reply:
done :)